daffodil-core/src/main/scala/org/apache/daffodil/dsom/ModelGroup.scala

```diff
@@ -74,17 +74,17 @@ object ModelGroupFactory {
         case <group>{ _* }</group> => {
           val pos = parent match {
             case ct: ComplexTypeBase => 1
-            case mg: ModelGroup => position
-            case gd: GlobalGroupDef => position
+            case mg: ModelGroup      => position
```

Comment:
Undo whitespace changes. Must have had an incorrect setting when this was last saved.